app/views/partials/theme/simple/items/form.liquid

```diff
@@ -82,6 +82,27 @@
     <button class="mt-8 button" value="{{ path }}">Submit</button>
   </form>
 
+  <div class="container dashboard w-1/2 px-4">
+    <p class="text-2xl text-gray-900 ">Reviews</p>
+    <div class="recent-reviews">
```

Comment:
You might want to take a look at https://tailwindcomponents.com/component/stars-review , no custom css rules should be needed to style this